warehouse/query-core/src/main/java/datawave/query/jexl/visitors/PushdownUnexecutableNodesVisitor.java

```diff
@@ -417,11 +392,6 @@ public Object visit(ASTMethodNode node, Object data) {
         return node;
     }
     
-    @Override
-    public Object visit(ASTSizeMethod node, Object data) {
```

Comment:
ASTSizeFunction?

Reply:
yes!  I will add it.

Reply:
Do we not want to visit the children of a size function?

Reply:
I guess I have to respond with a question. Can one of the children of the size function be an expression like maybe another function? I am thinking of something like f:func(bla).size()

Reply:
In the context of this function I guess it does not make sense as we were not doing it before.